README_specifications.md

```diff
@@ -0,0 +1,55 @@
+chitter-challenge specifications:
```

Comment:
This file should be the actual README.md. THis will ensure it shows up in your repo for all to see